simplq/src/components/pages/Admin/Item.jsx

```diff
@@ -48,7 +48,7 @@ function Item(props) {
     // Notifying in progress
     notificationButton = (
       <IconButton edge="end" color="primary" aria-label="notify">
-        <CircularProgress size={18} />
+        <LoadingIndicator />
```

Comment:
Will these trigger full screen loading?

Reply:
I've made it to show the indicator at the center of whatever component is rendering it. If it is the parent component of the page, it will be full screen, else it shows up at the center of just the part that's loading.